src/v/raft/group_configuration.h

```diff
@@ -128,6 +128,25 @@ class group_configuration final {
      */
     void discard_old_config();
 
+    /**
+     * Forcefully abort changing configuration. If current configuration in in
+     * joint state it drops the new configuration part and allow raft to operate
+     * with old quorum
+     *
+     * NOTE: may lead to data loss in some situations use only for cluster
+     * recovery from critical failures
+     */
+    void abort_configuration_change();
+
+    /**
+     * Reverts configuration change, the configuration is still in joint state
+     * but the direction of change is being changed
+     *
+     * NOTE: may lead to data loss in some situations use only for cluster
```

Comment:
Is this comment wrong?  I thought revert was supposed to be safe.

Reply:
yeah, revert is safe, need to update the comment